indico/htdocs/js/indico/modules/timetable/timetable/Management.js

```diff
@@ -682,19 +682,19 @@ type("RescheduleDialog", ["ExclusivePopupWithButtons"], {
 type("FitInnerTimetableDialog", ["ConfirmPopup"], {
 
     /**
-     * Returns the title of the timetable/session/entry
+     * Returns the title of the session block
      */
-    __getSessionTitle: function(){
-        return '"' + this.tt.contextInfo.title + '"';
+    __generateSessionBlockTitle: function(slotData) {
+        return '"' + slotData.title + (slotData.slotTitle ? ": " + slotData.slotTitle : '') + '"';
```

Comment:
I would use single quotes here: `": "` to use only one style of quoting within the same statement/line.